docs/INSTALL.md

```diff
@@ -1,50 +1,106 @@
-# Installation for lnd and btcd
+# Installation
 
-### If Glide isn't installed, install it:
+### Preliminaries
+  In order to work with [`lnd`](https://github.com/lightningnetwork/lnd), the following build dependencies are required:
+  
+  * **Go:** `lnd` is written in Go. To install, run one of the following commands:
+  
+    ```
+    # On Linux:
+    sudo apt-get install golang-go
 
+    # On Mac OS X
+    brew install go
+    ```
+    More detailed installation instructions can be found
+    [here](http://golang.org/doc/install). 
+
+    At this point, you should set your `$GOPATH` environment variable, which
+    represents the path to your workspace. By default, `$GOPATH` is set to
+    `~/go`. Be sure to set your `$GOPATH` every time you open a new terminal
+    window.
+    ```
+    export GOPATH=~/projects/lightning
```

Comment:
I figured that people may want to keep their Go workspaces separate. In any case, I implemented your above suggestion of recommending they put this step (as well as the `$GOPATH/bin` step below) into their .bashrc so I don't think it's an issue.